### DISCUSSION TITLE
3.6.8

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#76] [Announcements]
Blender 3.6.8, commit [f6ae60f3eeda](https://projects.blender.org/blender/blender/commit/f6ae60f3eeda).
Build environment:

    Visual Studio 2019, version 16.11.32
    Windows SDK 10.0.22621
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/3.6.8'>3.6.8</a>.</em>

**Comment by NightLancer** [comment]
I know that github is not that "chatty" place but I really wanted to thank you for your work! 
_I was postponing Blender for almost half a year, literally today asked about "which version is better to use"(without knowing it isn't supported on Windows7) - and here it is, exact needed version uploaded just 5 days ago._
